lib/merch_calendar/retail_calendar.rb

```diff
@@ -53,41 +50,39 @@ def end_of_month(year, merch_month)
 
     # Returns the date that corresponds to the first day in the merch week
     def start_of_week(year, month, merch_week)
-      week = MerchCalendar::MerchWeek.find(year, month, merch_week) 
-      week.start_of_week
+      start_of_month(year, month) + ((merch_week - 1) * 7)
     end
 
     # Returns the date that corresponds to the last day in the merch week
     def end_of_week(year, month, merch_week)
-      week = MerchCalendar::MerchWeek.find(year, month, merch_week) 
-      week.end_of_week
+      start_of_month(year, month) + (6 + ((merch_week - 1) * 7))
     end
 
     # Return the starting date for a particular quarter
     def start_of_quarter(year, quarter)
       case quarter
       when 1
-        start_of_month(year, 7)
+        start_of_month(year, 1)
```

Comment:
I'll acknowledge that this is scope creep, and I'm not asking it for this PR, but we should work towards limiting the amount of "magic" numbers in here. Magic + Dates = 😖